extra/modules/confiant-ad-quality/src/main/java/org/prebid/server/hooks/modules/com/confiant/adquality/model/RedisBidsData.java

```diff
@@ -0,0 +1,26 @@
+package org.prebid.server.hooks.modules.com.confiant.adquality.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.iab.openrtb.request.BidRequest;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Builder(toBuilder = true)
+@Value
+public class RedisBidsData {
+
+    BidRequest breq;
+
+    List<RedisBidResponseData> bresps;
+
+    public String toJson() {
+        try {
+            return new ObjectMapper().writeValueAsString(this);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
You are creating new `ObjectMapper` each time this method is called. 
1) This is heavy.
2) Its not data class responsibility to decide how it should be serialized.

Please, extract ObjectMapper creation and usage to the place where this method is called.